docs/launch-with/anvil-launch-with.rst

```diff
@@ -0,0 +1,61 @@
+AnVIL
+=====
+
+Dockstore integrates with the the Analysis, Visualization, and Informatics Lab-space (AnVIL) platform,
```

Comment:
Maybe a link would be helpful
https://www.genome.gov/Funded-Programs-Projects/Computational-Genomics-and-Data-Science-Program/Genomic-Analysis-Visualization-Informatics-Lab-space-AnVIL perhaps

Reply:
Added link